web-console/4-patterns/masthead.md

```diff
@@ -29,6 +31,32 @@ On the landing page, the only navigation available is the Primary Masthead.
 
 ## Implementation Details
 
+### User
+
+#### Regions
+- Users will have the ability to view and change their cluster region.
+
+![Full Masthead](img/Region.png)
+- For MVP, the cluster region will be listed as a fly-out option under the user menu in the masthead.  
+- The current region will be displayed in the dropdown.
+
+#### Set Home Page
```

Comment:
@beanh66 We may want to note that this setting will be saved per user per browser, so that QE & Documentation can have a better understanding of the feature.

Reply:
ehh ... i see that you have this in the documentation below, ignore me 💃

Reply:
@ncameronbritt I thought we decided to leave this region documentation as is for now until we firm up the new design with the context selector? I didn't add it as part of this PR, just kept it as is. Do you want me to remove it?